libs/live-config/src/featureFlags/defaultFeatures.ts

```diff
@@ -66,6 +66,8 @@ export const CURRENCY_DEFAULT_FEATURES = {
   currencyVelasEvm: DEFAULT_FEATURE,
   currencyCasper: DEFAULT_FEATURE,
   currencyNeonEvm: DEFAULT_FEATURE,
+  currencyLinea: DEFAULT_FEATURE,
+  currencyLineaGoerli: DEFAULT_FEATURE,
```

Comment:
To be discussed but shouldn't we cover both linea and lineagoerli with the same feature flag ?

Reply:
For the currencies currently supporting a testnet they have a different feature flag. I think it's better to have seperate feature_flag as goerli is only used in developer mode, its overridden locally by the developer when the mainnet is set in firebase for the users

Reply:
"feature_flag as goerli is only used in developer mode"

=> that's why i'm asking, we are already out of the standard usage